bindings/pyroot/pythonizations/src/RVecPyz.cxx

```diff
@@ -20,6 +20,7 @@
 
 ////////////////////////////////////////////////////////////////////////////
 /// \brief Adopt memory of a Python object with array interface using an RVec
+/// \param[in] self self object
```

Comment:
Same here `self Always null, since this is a module function.`

Reply:
done